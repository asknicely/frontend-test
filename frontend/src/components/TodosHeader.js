import React from 'react'

const Header = (props) => {
    return (
        <tr>
            <th>#</th>
            <th>User</th>
            <th>Description</th>
            <th>Status</th>
            <th></th>
        </tr>
    )
}

export default Header