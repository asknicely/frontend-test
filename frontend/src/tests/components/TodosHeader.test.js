import React from 'react';
import { shallow } from 'enzyme';
import TodosHeader from '../../components/TodosHeader';

test('should render TodosHeader correctly', () => {
	const wrapper = shallow(
		<TodosHeader />
	);
	expect(wrapper).toMatchSnapshot();
});
