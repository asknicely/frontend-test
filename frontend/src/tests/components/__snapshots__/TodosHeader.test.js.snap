// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TodosHeader correctly 1`] = `
<tr>
  <th>
    #
  </th>
  <th>
    User
  </th>
  <th>
    Description
  </th>
  <th>
    Status
  </th>
  <th />
</tr>
`;
